sample-apps/java-sample-app/auto/build.gradle.kts

```diff
@@ -33,20 +32,42 @@ repositories {
     mavenCentral()
 }
 
+jib {
+    from {
+        image= "eclipse-temurin:17"
+    }
+    to {
+        image = "java-auto-instrumentation-sample-app"
+    }
+    extraDirectories {
+        paths { 
+            path {
+                setFrom("$buildDir/javaagent")
+            }
+        }
+    }
+    container {
+        ports = listOf("8080")
+        jvmFlags = listOf(
+            "-javaagent:aws-opentelemetry-agent-1.23.0.jar",
+            "-Dotel.javaagent.extensions=${buildDir}/javaagent/extension.jar")
+    }
+}
+
 dependencies {
     // Base application
     implementation(project(":base"))
 
     // Necessary to download the jar of the Java Agent
-    javaagentDependency("software.amazon.opentelemetry:aws-opentelemetry-agent:${otelVersion}@jar")
+    javaagentDependency("software.amazon.opentelemetry:aws-opentelemetry-agent:1.23.0@jar")
```

Comment:
we can remove the `@jar` since we fixed the issue with the default artifact.

Reply:
Removed in my latest commit.